src/main/java/org/rulez/demokracia/pdengine/VoteFilter.java

```diff
@@ -0,0 +1,19 @@
+package org.rulez.demokracia.pdengine;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public interface VoteFilter {
+
+	public default List<CastVote> filterVotes(final List<CastVote> votes, final String assurance) {
```

Comment:
https://sonarcloud.io/project/issues?id=org.rulez.demokracia%3APDEngine&open=AWmzxhLb5QE9aC-CRoAN&pullRequest=289&resolved=false

Reply:
That's why I don't like smart beans. I would prefer to refactor all the business logic to services from Vote and similar data objects. What is your opinion about it?

Reply:
Let's try it.